src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1057,5 +1057,75 @@ public override string VisualizeLocalType(object type)
         }
 
         #endregion
+
+        #region Span
+        //PROTOTYPE(span): this will be updated when rules for defining span are implemented
+        //                 most likely we would just pick the actual binary/corlib where
+        //                 span lives.
+        protected static CSharpCompilation CreateCompilationWithMscorlibAndSpan(string text, CSharpCompilationOptions options = null, CSharpParseOptions parseOptions = null)
+        {
+            var reference = CreateCompilation(
```

Comment:
>var [](start = 12, length = 3)

Can we cache the span sources so we don't have to compile them over and over again?